packages/examples/src/scenes/canvas-operations-filters-stacking.tsx

```diff
@@ -0,0 +1,139 @@
+import {makeScene2D} from '@motion-canvas/2d';
+import {AnimatedMotionCanvasIcon} from '../components/AnimatedMotionCanvasIcon';
+import {createSignal} from '@motion-canvas/core/lib/signals';
+import {linear} from '@motion-canvas/core/lib/tweening';
+import {contrast, saturate} from '@motion-canvas/2d/lib/partials';
+import {
+  Circle,
+  Grid,
+  Layout,
+  Rect,
+  Txt,
+  TxtProps,
+} from '@motion-canvas/2d/lib/components';
+import {CodeBlock} from '@motion-canvas/2d/lib/components/CodeBlock';
+
+export default makeScene2D(function* (scene) {
+  const t = createSignal(0);
+  scene.add(<Rect size={5000} fill={'#111'} />);
+
+  const createTextValue = (val: number, leftCount = 1, rightCount = 2) => {
+    const value = String(Math.round(val * 100) / 100);
+
+    let [left, right] = value.split('.');
+    if (!right) {
+      right = '';
+    }
+
+    left = left.padStart(leftCount).substring(left.length - leftCount);
+
+    if (rightCount === 0) {
+      return left;
+    }
+
+    right = right.padEnd(rightCount, '0').substring(0, rightCount);
+    return `${left}.${right}`;
+  };
+
+  const commonTxtValues: Partial<TxtProps> = {
+    fontFamily: 'monospace',
+    fill: 'white',
+    fontSize: 40,
+  };
+  const commonTxtValuesSmall: Partial<TxtProps> = {
+    fontFamily: 'monospace',
+    fontSize: 35,
+    fontWeight: 600,
+  };
+
+  const saturateValue = createSignal(1);
+  const contrastValue = createSignal(1);
+
+  scene.add(
+    <Layout position={[-600, -200]}>
+      <AnimatedMotionCanvasIcon
+        filters={[saturate(saturateValue), contrast(contrastValue)]}
+        timePassed={t}
+      />
+      <CodeBlock
```

Comment:
This also looks off:
![image](https://user-images.githubusercontent.com/64662184/223300518-ab896569-d696-4b48-bcaf-4c66bc752abb.png)
The font size shouldn't be different than the one in the CodeBlock